CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contribution Guidelines
+
+We always appreciate contributions in the form of pull requests. In this
+document, we provide a list of conventions and practices that we expect
+`TypeTopology` contributors to follow.
+
+- Developments are organised using subdirectories inside the `source` folder.
+  For example, the domain theory development lives in `source/DomainTheory`.
+- The file `index.lagda` contains a global index for `TypeTopology`, importing
+  the index for each development in each directory (e.g. `DomainTheory.index`).
+  If you are starting a new development, make sure that it has an `index.lagda`
+  file and that it is imported in the global `index.lagda` file.
+- The indentation length used in `TypeTopology` is **a single space**. Please be
+  careful about this as it is a nonstandard convention.
+- `TypeTopology` does not use the level notation of Agda and instead provides a
+  wrapper (in module [`MLTT.Universes`][1]) around this to keep the notation
+  close to pen-and-paper conventions. We use the variables `𝓤 𝓥 𝓦` for universe
+  levels and use the notation `𝓤  ̇` to denote the universe at level `𝓤`. Please
+  make sure that your code uses this and avoids the level notation.
```

Comment:
This could be a little clearer: 

- I assume by the 'level notation' you mean the use of things like `Type l`?
- We can also help the reader out a bit and instruct them that to type that little dot after a universe, they should type `space-backlash-caret-period`.

Reply:
Agreed! I'll finish this PR up now and will integrate these changes.